Jenkinsfile

```
@@ -1,44 +1 @@
-/*
- * Copyright 2019 Google Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-pipeline {
-    agent none
-
-    environment {
-        GOOGLE_BUCKET = "${GCS_IT_BUCKET}"
-        GOOGLE_PROJECT_ID = "${GCS_IT_PROJECT_ID}"
-    }
-
-    stages {
-        stage("Build and test") {
-            agent {
-                kubernetes {
-                    cloud 'kubernetes'
-                    label 'maven-pod'
-                    yamlFile 'jenkins/maven-pod.yaml'
-                }
-            }
-            steps {
-                container('maven') {
-                    withCredentials([[$class: 'StringBinding', credentialsId: env.GCS_IT_CRED_ID, variable: 'GOOGLE_CREDENTIALS']]) {
-                        // build
-                        sh "mvn verify -ntp -DskipIts=false"
-                    }
-                }
-            }
-        }
-    }
-}
+buildPlugin()
```

```suggestion
buildPlugin(configurations: buildPlugin.recommendedConfigurations())
```
this would be great so that the plugin is built against JDK 8 and 11.